labs/lab4/cafe.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: coffee
-        image: nginxinc/ingress-demo
+        image: chrisakker/ingress-demo
```

Comment:
This needs to revert back to nginxinc/ingress-demo when we fix the "gray box is missing and text is not centered on latest image and fav icon missing" issue in latest image.